scripts/migrate_to_fork.py

```diff
@@ -324,6 +324,10 @@ def setup_remotes(username: str, fork_url: str, auto_accept: bool = False) -> st
     """
     print_step(4, "Setting up git remotes")
 
+    # this will remove tracking branches corresponding to branches on origin whose names
+    # differ only in case:
```

Comment:
```suggestion
    # This will sync the local references with the upstream ones and delete refs to branches that
    # don’t exist anymore on upstream repos.
    # In particular, this will ensure the branches with duplicate names up to case that were recently
    # deleted on the main repository do not cause trouble in the migration.
```

This is a little more precise about what is achieved.